db/seeds.rb

```diff
@@ -72,43 +82,81 @@
 
 posts = [
   {
-    post_text: 'Legend has it that the 1st cheese was created 4,000 years ago, accidentally!',
+    post_text: 'Why did the two golfer bring two pairs of pants?..... In cse he got a hole in one!',
     group_id: 1
   },
   {
-    post_text: 'It takes about 10 pounds of milk to make just 1 pound of cheese!',
+    post_text: 'I wrote a song about a tortilla.... Actually, it\'s more of a wrap.',
     group_id: 1
   },
   {
-    post_text: 'Just found out some cheeses are illegal in the United States.',
+    post_text: 'I saved a bunch of money by switching over to geico',
+    group_id: 1
+  }, 
+  {
+    post_text: 'What is red and really bad for your teeth?.... A flying brick!',
     group_id: 1
   },
   {
-    post_text: 'Many lactose intolerant people can eat cheese!',
+    post_text: 'Better grab a dubmrella... It\'s really stupid out there today.',
     group_id: 1
   },
   {
-    post_text: 'Who knew null was an object?',
+    post_text: 'What do you call a pig that does karate?..... A pork chop!',
+    group_id: 1
+  },
+  {
+    post_text: 'The past, present, and future walked into a bar.... it was tense',
+    group_id: 1
+  },
+  {
+    post_text: 'My boss told me to have a good day..... So I went home.',
+    group_id: 1
+  },
+  {
+    post_text: 'Love is like a fart, if you have to forget it, it\'s probably shit',
+    group_id: 1
+  },
+  {
+    post_text: 'Legend has it that the 1st cheese was created 4,000 years ago, accidentally!',
     group_id: 2
   },
   {
-    post_text: 'NaN is actually a number (typeof NaN > "number") and (NaN === NaN > false). How about that?',
+    post_text: 'It takes about 10 pounds of milk to make just 1 pound of cheese!',
     group_id: 2
   },
   {
-    post_text: 'Reminder to everyone, 0 evaluates to false!',
+    post_text: 'Just found out some cheeses are illegal in the United States.',
+    group_id: 2
+  },
+  {
+    post_text: 'Many lactose intolerant people can eat cheese!',
     group_id: 2
   },
+  {
+    post_text: 'Who knew null was an object?',
+    group_id: 3
+  },
+  {
+    post_text: 'NaN is actually a number (typeof NaN > "number") and (NaN === NaN > false). How about that?',
+    group_id: 3
+  },
+  {
+    post_text: 'Reminder to everyone, 0 evaluates to false!',
+    group_id: 3
+  },
   {
     post_text: "El Cotixan for some bomb Mexican food if your're in the El Cajon area",
-    group_id: 4
+    group_id: 5
   },
   {
     post_text: 'Has anyone tried Menya Ultra in Kearny Mesa? Is it any good?',
-    group_id: 4
+    group_id: 5
```

Comment:
Typically you would let rails handle assigning this id. It looks like this is being created on line 175.